src/Home/index.js

```diff
@@ -1,16 +1,132 @@
-import React, { Component } from 'react';
-import { View, Text, AsyncStorage, Linking, Platform } from "react-native";
-import commonStyle from '../Styles/common.style';
+import React, { Component } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  Image,
+  ScrollView,
+  FlatList
+} from 'react-native';
+import { Query } from "react-apollo";
+import commonStyle from "../Styles/common.style";
+import themeStyle from "../Styles/theme.style";
+import loader from '../Assets/Images/double-ring-loader.gif';
+import Java_developers from '../graphQL_Queries/getAllJavadevelopers';
+import { TouchableOpacity } from 'react-native-gesture-handler';
 class Home extends Component {
```

Comment:
Remember to leave a space between the imports and the logic.

Reply:
I've updated this. But I don't know why it still shows it.

Reply:
oh. It's okay now. I can see a space